src/config/vscode.config.json

```diff
@@ -1,6 +1,12 @@
 {
   "ide_path": "",
   "plugin_cache_path": "",
+  "project_paths" :
```

Comment:
please change the file name to vscode_config.json

Reply:
@sshveta , Renamed the file